### ISSUE TITLE
modinfo: ERROR: Module nvidia not found.

### ISSUE THREAD

**Issue by woeful1** [Bumblebee-Project/Bumblebee#775]
Hello 
I fallow this [https://fedoraproject.org/wiki/Bumblebee](https://fedoraproject.org/wiki/Bumblebee) to install  bumblebee and i have this 
`
[phoenix@localhost Fallout 4]$ bumblebee-nvidia --check
**Warning! This NVIDIA driver userland
 /usr/lib64/nvidia-bumblebee/libGL.so.1 library is missing!

Warning! nvidia.ko was not created in in the kernel tree!
modinfo output for NVIDIA:

modinfo: ERROR: Module nvidia not found.**

Check bbswitch kernel module...

bbswitch is loaded into the current kernel ok.

All NVIDIA checks completed, but there were 1 or more failures...
Try running this script with the --debug option to find clues about what has
gone wrong with the NVIDIA driver compile process.
`
and this 

`
**[phoenix@localhost Fallout 4]$ uname -a**
Linux localhost.localdomain 4.2.3-300.fc23.x86_64 #1 SMP Mon Oct 5 15:42:54 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
**[phoenix@localhost Fallout 4]$ rpm -q kernel-devel**
kernel-devel-4.3.3-303.fc23.x86_64
kernel-devel-4.2.3-300.fc23.x86_64
`
and this 

`
**[phoenix@localhost Fallout 4]$ optirun -vv glxgears**
[  550.272307] [DEBUG]Reading file: /etc/bumblebee/bumblebee.conf
[  550.272436] [INFO]Configured driver: nvidia
[  550.272462] [ERROR]The Bumblebee daemon has not been started yet or the socket path /var/run/bumblebee.socket was incorrect.
[  550.272466] [DEBUG]Socket closed.
[  550.272472] [ERROR]Could not connect to bumblebee daemon - is it running?
`

and this 

`
**[phoenix@localhost Fallout 4]$ bumblebeed**
[  590.585981] [ERROR]Module 'nvidia' is not found.
`

and finaly bellow command return nothing

`
[phoenix@localhost Fallout 4]$ sudo /usr/sbin/bumblebee-nvidia
`

Thanks for your help.

**Comment by gsgatlin** [comment]
Make sure you have the rpm package "kernel-devel" installed. (dnf install kernel-devel)

Make sure you have "secure boot" disabled in yoir UEFI or BIOS.

Past the output from 

sudo bumblebee-nvidia --debug

to a site such as http://pastebin.com/ or https://paste.fedoraproject.org  and post a link to that here.

**Comment by woeful1** [comment]
Hello thanks for your rplay.

    [phoenix@localhost ~]$ sudo dnf install kernel-devel
    Package kernel-devel-4.3.3-303.fc23.x86_64 is already installed, skipping.
    Package kernel-devel-4.2.3-300.fc23.x86_64 is already installed, skipping.
    Dependencies resolved.
    ==============================================================================================================================================================
     Package                                 Arch                              Version                                   Repository                          Size
    ==============================================================================================================================================================
    Skipping packages with conflicts:
    (add '--best --allowerasing' to command line to force their upgrade):
     kernel-devel                            x86_64                            4.4.9-300.fc23                            updates                             10 M

    Transaction Summary
    ==============================================================================================================================================================

    Nothing to do.
    Complete!

I check the bios and secure boot is disabled.and 

sudo bumblebee-nvidia --debug

just return 

    --debug mode selected.

There is one thing that I should mention it. When i first install bummlebee i think i should install nvida driver sepratly then i download nvidia driver  from the nvidia website and install it .After install nvida driver display manager not loaded and i uninstall the nvidia driver with "nvidia-uninstall" . These can cause the problem?

**Comment by gsgatlin** [comment]
Yeah. You want to make sure you start with a system that has no nvidia drivers installed from elsewhere. That is probably what the problem is.

I do not know what it means when you have no output from bumblebee-nvidia --debug. It should print a bunch of dots.
